learning/tour-of-beam/frontend/lib/pages/welcome/screen.dart

```diff
@@ -216,30 +218,60 @@ class _IntroText extends StatelessWidget {
           color: BeamColors.grey2,
           constraints: const BoxConstraints(maxWidth: _dividerMaxWidth),
         ),
-        RichText(
-          text: TextSpan(
-            style: Theme.of(context).textTheme.bodyLarge,
-            children: [
+        const _IntroTextBody(),
+      ],
+    );
+  }
+}
+
+class _IntroTextBody extends StatelessWidget {
+  const _IntroTextBody();
+
+  void _openLoginDialog(BuildContext context) {
```

Comment:
There's a habit to put methods in the order of usage. This is called in `build`, so should be below it.